src/scaffoldmaker/utils/annulusmesh.py

```diff
@@ -410,22 +410,28 @@ def createAnnulusMesh3d(nodes, mesh, nextNodeIdentifier, nextElementIdentifier,
                 eft1 = eftFactory.createEftNoCrossDerivatives()
                 # work out if scaling by global -1
                 scaleMinus1 = mapStartLinearDerivativeXi3 or mapEndLinearDerivativeXi3
-                if (not scaleMinus1) and startDerivativesMap:
+                if (not scaleMinus1) and mapStartDerivatives:
                     for n3 in range(2):
-                        for i in range(2):
-                            derivativesMap = startDerivativesMap[n3][e1] if (i == 0) else startDerivativesMap[n3][en]
-                            for map in derivativesMap:
-                                if map and (-1 in map):
-                                    scaleMinus1 = True
-                                    break
-                if (not scaleMinus1) and endDerivativesMap:
+                        # need to handle 3 or 4 maps (e1 uses first 3, en uses last 3)
```

Comment:
Correct comment - should say "e1 uses last 3, en uses first 3"

Reply:
Fixed.